### ISSUE TITLE
Auto Heirlooms not keeping radiants

### ISSUE THREAD

**Issue by aparod** [Zorn192/AutoTrimps#67]
I have Auto Heirlooms enabled and set it to keep all radiating heirlooms.  I had 5 or 6 radiating heirlooms already being carried, went AFK for a while, and when I came back, all but one of my radiating heirlooms had been replaced with plagued.

**Comment by Zorn192** [comment]
Probably because the plagues were so much better. If its below the rarity you have selected it divide its value by 3. If its at or above its multiplied by 3. Also, its keeping them for me perfectly, so perhaps look at the mods you have selected.

**Comment by aparod** [comment]
The only options I have enabled right now are Auto Heirlooms > All > Radiating.  The others (Auto Nu, Shields, Staffs, Cores) are all disabled.

When hovering over the rarity dropdown it says, "Keeps the selected rarity of heirloom, recycles all others".  That isn't what's happening for me, so either there's a bug or the description should be updated to describe the actual behavior.

I have had perfect plagued heirlooms since shortly after they were released and I have no use for more.  I want autotrimps to stop filling my carry space with plagued and only keep the radiants.  Maybe I'd have better luck if I specified the mods I want, like you have done.
